presto-main/src/test/java/io/prestosql/operator/scalar/TestMathFunctions.java

```diff
@@ -1142,42 +1142,42 @@ public void testGreatest()
         assertFunction("greatest(TINYINT'-1', TINYINT'-2')", TINYINT, (byte) -1);
         assertFunction("greatest(TINYINT'5', TINYINT'4', TINYINT'3', TINYINT'2', TINYINT'1', TINYINT'2', TINYINT'3', TINYINT'4', TINYINT'1', TINYINT'5')", TINYINT, (byte) 5);
         assertFunction("greatest(TINYINT'-1')", TINYINT, (byte) -1);
-        assertFunction("greatest(TINYINT'5', TINYINT'4', CAST(NULL AS TINYINT), TINYINT'3')", TINYINT, (byte) 5);
+        assertFunction("greatest(TINYINT'5', TINYINT'4', CAST(NULL AS TINYINT), TINYINT'3')", TINYINT, null);
```

Comment:
You do not even need to add a new test, nice.

Reply:
I'm not sure what this comment means.